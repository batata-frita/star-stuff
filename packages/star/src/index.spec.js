const star = require('./index')

describe('lifespan', () => {
  it('should generate a lifespan between 0 and 1', () => {
    expect(star.lifespan(0.5)).toMatchSnapshot()
  })
})

describe('type', () => {
  it('should generate a type', () => {
    expect(star.type(0.7)).toMatchSnapshot()
  })
})
