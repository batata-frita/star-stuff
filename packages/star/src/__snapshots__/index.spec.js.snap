// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lifespan should generate a lifespan between 0 and 1 1`] = `0.39277242636308074`;

exports[`type should generate a type 1`] = `
Object {
  "name": "G",
  "threshold": 0.6,
}
`;
